camera_matrix: [915.828857, 0.000000, 644.015094, 0.000000,
0.000000, 925.069885, 372.912666, 0.000000,
0.000000, 0.000000, 1.000000, 0.000000,]

rlc:  [   0.999715  ,-0.00610437 ,0.023068,  
  0.0113064 , 0.972496   ,-0.232646,  
 -0.0210134 , 0.23284    ,0.972288 ]

tlc: [ -0.0549389263, -0.059792, 0.12]






